### DISCUSSION TITLE
Due to PHP limitations, the fork driver may not be used within web requests.

### DISCUSSION THREAD

**Discussion by LfJohnVo** [spatie/fork#62] [Q&A]
have this error: Due to PHP limitations, the fork driver may not be used within web requests.

and i dont know why

**Comment by ChrisSantiago82** [comment]
❗️ [pcntl](https://www.php.net/manual/en/intro.pcntl.php) only works in CLI processes, not in a web context. ❗️ [posix](https://www.php.net/manual/en/book.posix.php) required for correct handling of process termination for Alpine Linux.
